tests/functional/controllers/test_schedule_controller.py

```diff
@@ -69,6 +45,8 @@ def test_get_list_of_schedules(tables, client):
 
     client.post(ENDPOINT, headers=HEADERS, data=json.dumps({'hourStart': '8:00', 'hourEnd': '16:00',
                                                             'scheduleDays': ['Monday']}))
+    assert resp.status_code == HTTPStatus.OK
```

Comment:
> tests/functional/controllers/test_schedule_controller.py::test_get_list_of_schedules requires fixing - @matpiotrowski, any ideas?

The user has no permission to create a schedule here and resp variable is not assigned on line 46 so the forbidden error is not detected on the next line.

Reply:
Thanks! Fixed in 1125df9d5cc3583badb45fdcc916a13c14ac99dc